src/client/datascience/constants.ts

```diff
@@ -2,557 +2,10 @@
 // Licensed under the MIT License.
 'use strict';
 
-import * as path from 'path';
 import { EXTENSION_ROOT_DIR, JVSC_EXTENSION_ID } from '../common/constants';
+import * as path from 'path';
 
-export const DefaultTheme = 'Default Light+';
```

Comment:
Moved all of these to datascience-ui/constants so they can be referenced in the UI parts without referencing node constructs (webpack no longer auto fills all node methods)